apps/quiz-for-america/index.js

```diff
@@ -109,6 +109,40 @@ app.intent('AMAZON.StopIntent', function(request, response) {
     response.say(say.join('\n'));
 });
 
+app.intent('QuizMeOnIntent',
+    {
+    "slots":{"QUESTION_NUMBER":"NUMBER"},
+    "utterances":[ "quiz me on question {1-200|QUESTION_NUMBER}" ]
+    },
+    function(request, response) {
+      var session = request.sessionDetails.attributes;
+      var questionNumber = request.slot('QUESTION_NUMBER');
+      if (quiz.isAskable(questionNumber)) {
+        // provide the question
+        var q = quiz.getQuestion(questionNumber);
+        var say =["<s>Ok, here's question " + questionNumber + " <break strength=\"medium\" /></s>"];
+        response.shouldEndSession(false, 'What do you think? Is it '+q.choices()+'?');
+        say.push(q.questionAndAnswers());
+        var myResponse = say.join('\n');
+        response.say(myResponse);
+        session.q = q.id;
+      } else {
+        var say ="<s>Sorry, I don't know question " + questionNumber + " <break strength=\"medium\" /></s>";
+        response.shouldEndSession(false, 'Ask me for another quiz or to quiz you on another question number.');
+        response.say(say);
+        session.q = null;
+      }
+
+      // session management
+      Object.keys(session).forEach((key) => {
+          response.session(key, session[key]);
+      });
+      app.db.saveSession(request.userId, session).then(() => {
+          console.log('saved session');
+          response.send();
+      });
+});
```

Comment:
since this is mainly for testing/debugging, I don't think we want to save the question to the session, because that will prevent this user from getting it again in a regular quiz.